composer.json

```diff
@@ -46,6 +46,7 @@
             "DDTrace\\Tests\\": "./tests/"
         }
     },
+    "minimum-stability": "dev",
```

Comment:
I am trying to figure it out consequences of this. So far it seems ok to me, since [`minimum-stability`](https://getcomposer.org/doc/04-schema.md#minimum-stability) is marked as "root-only" and "root-only" properties of dependencies [seems to be ignored](https://getcomposer.org/doc/04-schema.md#root-package).

Anyone has more knowledge of this matter and can confirm?

Reply:
I looked the consequences of that up as well, but seemed fine to me too.